.github/workflows/dispatch-wgc-homebrew-bump.yml

```
@@ -0,0 +1,55 @@
+permissions: {}
+
+jobs:
+  dispatch:
+    name: Dispatch wgc-release if cooldown cleared
+    runs-on: ubuntu-latest
+    timeout-minutes: 5
+
+    steps:
+      - name: Authenticate to homebrew-cosmo
+        id: app-token
+        uses: actions/create-github-app-token@v3
```

actionlint (1.7.12)

[error] missing input "app-id" which is required by action "actions/create-github-app-token@v3". all required inputs are "app-id", "private-key"

(action)